iroha/src/genesis.rs

```diff
@@ -83,7 +82,7 @@ impl GenesisNetwork {
                     .transactions
                     .iter()
                     .map(|raw_transaction| {
-                        raw_transaction.sign_and_accept(&genesis_key_pair, max_instructions_number)
+                        raw_transaction.sign_and_accept(&genesis_key_pair, usize::MAX)
```

Comment:
Different machines will put different values here. Are we ok with that?

Reply:
I thought that we could make an exception to genesis block, as we trust our self-made json with transactions.

Reply:
Though I agree that genesis block is a special case, for consensus simplicity I suggest to try to have the minimal difference for it. So to my mind there also should be a limit for instructions as for any other block.

TL;DR I suggest not to change it here.